src/Core/Search/Discussion/Gambit/AuthorGambit.php

```diff
@@ -50,7 +51,7 @@ protected function conditions(AbstractSearch $search, array $matches, $negate)
 
         $ids = [];
         foreach ($usernames as $username) {
-            $ids[] = $this->users->getIdForUsername($username);
+            $ids[] = is_numeric($username) ? $username : $this->users->getIdForUsername($username);
```

Comment:
OK, I agree. Then we can leave this PR for now until we decide to implement the new validation and figure out the migration strategy.